apps/token-e2e/src/integration/flow/governance-flow.cy.js

```diff
@@ -1103,53 +1008,63 @@ context(
           .should('be.visible');
       });
 
-      it('Unable to create a freeform proposal - when json parent section contains unexpected field', function () {
+      // Json containing unexpected field no longer fails submission
+      it.skip('Unable to create a freeform proposal - when json parent section contains unexpected field', function () {
```

Comment:
question (non blocking): I don't have full understanding of whats going on with these failed tests, but if this is functionality that no works differently lets just remove the tests

Reply:
Normally I would agree but these tests are testing an AC which might not be needed anymore because of the form validations. I think it's best to keep them for now until we get that AC changed/removed or find a different way to test this.